src/main/scala/ai/privado/languageEngine/go/processor/GoProcessor.scala

```diff
@@ -57,6 +58,8 @@ object GoProcessor {
             s"${TimeMetric.getNewTime()} - Run oss data flow is done in \t\t\t- ${TimeMetric.setNewTimeToLastAndGetTimeDiff()}"
           )
 
+          new SQLParser(cpg, sourceRepoLocation, ruleCache).createAndApply()
+          new SQLQueryParser(cpg).createAndApply()
```

Comment:
Are we also calling SQLQueryTagger in Go's PrivadoTagger?

Reply:
This will help in tagging SQL nodes which match sources regex

Reply:
It wasn't added previously, addressed(added) in commit: ab05f8ed6487167bcdae4e683e8f4be680fd4f50